code/modules/mapping/map_template.dm

```diff
@@ -37,6 +38,7 @@
 	for(var/L in turfs)
 		var/turf/B = L
 		atoms += B
+		areas |= get_area(B)
```

Comment:
Juse use turf.loc, no need to search for it.

Reply:
B.loc in this case